pdf/pdfdocument.cpp

```diff
@@ -108,6 +108,10 @@ PDFDocument::LinkMap PDFDocument::linkTargets() const
 {
     return d->thread->linkTargets();
 }
+const PDFDocument::TextList PDFDocument::textBoxesAtPage(int page)
```

Comment:
Empty line as method separator.

A bit personal preference, but I think non-pointer/reference as const return type are pointless. Caller can assign to const variable if that seems fit.